### ISSUE TITLE
Subdomain redirection

### ISSUE THREAD

**Issue by SilverCory** [minecraftly/redirect#4]
This isn't happening with that Apache rewrite

**Comment by viet** [comment]
Yeah currently with the new website, the custom player page feature isn't there anymore. So I guess we'll redirect like this for now.

**Comment by SilverCory** [comment]
Ahh okay :P
